karrot/history/tests/test_models.py

```diff
@@ -35,3 +35,12 @@ def test_activity_left_late(self):
         self.assertEqual(annotated.activity_leave_seconds, leave_seconds_before_activity)
         self.assertEqual(History.objects.activity_left_late(seconds=leave_seconds_before_activity).count(), 1)
         self.assertEqual(History.objects.activity_left_late(seconds=leave_seconds_before_activity - 1).count(), 0)
+
+
+class HistoryTypusTests(TestCase):
+    def test_typus_ids_are_unique(self):
```

Comment:
+1 for test :) ... I thought about it before, it's an easy mistake to make, and I'm sure has happened before...